### PR TITLE
Add Headline component

### PR DESCRIPTION
**Pull request by jtart**
_Creates a Headline component with a headline block as its prop. The component reaches into the block, extracts the `text` property, and passes it to a `h1` element._

* [x] Fixes https://github.com/bbc/simorgh/issues/116
* [x] Up-to-date with latest - `git pull --rebase origin latest`
* [x] Runs locally - `npm run dev` & [http://localhost:7080/](http://localhost:7080/)
* [x] Tests added for new features
* [x] Tests pass - `npm test`
* [x] E2E tests pass - `npm run test:e2e` (requires server running see [README](https://github.com/bbc/simorgh#tests) for details)

* [ ] Test engineer approval

### CODE REVIEW

**Comment by jtart** [comment]
Travis and codeclimate diff coverage is not running on this PR (not working on forked repos), but tests are passing locally and test coverage has increased by 2%.

**Review by sareh — DISMISSED** [review]
Looks great!

File: src/app/components/Headline/index.jsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+const Headline = ({ blocks }) => {
+  const { text } = blocks[0].model.blocks[0].model;
```
Comment (pjlee11) [inline src/app/components/Headline/index.jsx:5]: Currently this doesn't check for the existence of each key within the object. Therefore if any of the chained key's do not exist we'd come up against an "cannot read property of undefined" error. As discussed yesterday we will most likely need to create a recursive method at some point to handle this type of tree walk. However in the mean time, as we create component's tightly coupled to the blocks in the feed, we could use something like lodash's [get](https://lodash.com/docs/4.17.4#get) to ensure this return either an `undefined` or the default value instead of potentially throwing a "cannot read property of undefined" error.

Comment (pjlee11) [reply src/app/components/Headline/index.jsx:5]: This was discussed in our group channel - for now as we are using static data we can ensure that the structure is correct and this reduces the risk on "cannot read property of undefined" errors. If we get to a stage where we are using live data feeds, which could be malformed, and haven't changed the way we traverse the tree I'd suggest we use lodash get method and import it in isolation as described here: 
https://stackoverflow.com/questions/35250500/correct-way-to-import-lodash 
TLDR: `import has from 'lodash/has';` only loads the get method into your application

**Review by pjlee11 — APPROVED** [review]
Comments left for future reference.

LGTM 👍

**Review by sareh — APPROVED** [review]
